### ISSUE TITLE
Docker-setup: 500 for 2FA-settings

### ISSUE THREAD

**Issue by unicode-it** [mosparo/mosparo#265]
### Version

Latest docker image

### Reproduce

- Start and setup application
- go to: /account/two-factor/start

### Error

```
Oops! An Error Occurred
The server returned a "500 Internal Server Error".
Something is broken. Please let us know what you were doing when this error occurred. We will fix it as soon as possible. Sorry for any inconvenience caused.
```

Is there any option to turn debugging on for the docker container?

**Comment by zepich** [comment]
Hi @unicode-it 

Thank you very much for your message. I'm sorry for that issue.

mosparo usually logs the error in the log file. I suggest that you connect to your Docker container with a command like this:

```bash
docker exec -it mosparo-docker-mosparo_web-1 /usr/bin/bash
```
_(Adjust the name `mosparo-docker-mosparo_web-1` to the name your container has)_

And then output the error log file:

```bash
cat /mosparo-data/var/log/prod-2024-08-14.log
```

Please let me know what error you see there.

Kind regards,

zepich

**Comment by unicode-it** [comment]
Thank you for the instructions! I find this:

```
[2024-08-14T11:35:32.559003+00:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "Issuer must not contain a colon." at ParameterTrait.php line 177 {"exception":"[object] (InvalidArgumentException(code: 0): Issuer must not contain a colon. at /mosparo/vendor/spomky-labs/otphp/src/ParameterTrait.php:177)"} []
```

**Comment by unicode-it** [comment]
Maybe I have to configure something somewhere first? But anyway this exception should be handled.

**Comment by zepich** [comment]
Hi @unicode-it 

Thank you very much for your response and the output of your log file.

Could you have used a colon (:) in the name of the mosparo installation?

![image](https://github.com/user-attachments/assets/b495d7c7-19a8-4127-b897-e340b20a0357)

If yes, please remove it and try again. It looks like that is the issue. If that solves the issue, we will add the required constraints so that this situation can not happen again.

Kind regards,

zepich

**Comment by unicode-it** [comment]
Thank you for the blazingly fast response. It solved the issue! Our company name contains a colon. 

Another good change for the docker container would be to log errors to stdout/stderr.

**Comment by unicode-it** [comment]
Another of topic question:
We created a helm chart to deploy mosparo in k8s. We didn't find any. Would you like us to contribute it?

**Comment by unicode-it** [comment]
A helmchart is a folder with several template files to create k8s manifests based on a values file. 
I could either create a repo in our namespace and just put it there or make a pull request on your mosparo repo, put it in a subfolder "helm" and hand it over under your maintenance (like docker). It could then be mentioned in the docs as another way of installation.

**Comment by zepich** [comment]
Hi @unicode-it 

Okay, perfect; thank you for the explanation.

I suggest we create a new repo, `mosparo/helmchart`, and you make a pull request for that repo.

Would that be okay for you?

Kind regards,

zepich

**Comment by unicode-it** [comment]
Yes perfect

**Comment by zepich** [comment]
Hi @unicode-it 

Awesome! Here we go: https://github.com/mosparo/helmchart

I set MIT as a license as all the other repositories, but please let me know if you prefer a different license.

Thank you very much for your help!

Kind regards,

zepich

**Comment by zepich** [comment]
The main issue was fixed with v1.2.6.
Logging to file and `stderr` was also optimized with v1.2.6.
